values/backend-org-1.yaml

```diff
@@ -89,9 +89,11 @@ extraEnv:
 channels:
   - mychannel:
       chaincode:
+        restricted: false
```

Comment:
It looks like there is an extra indentation here as it is the channel which is restricted and not the chaincode :)